e2e-app/src/app/timepicker/navigation/timepicker-navigation.e2e-spec.ts

```diff
@@ -0,0 +1,87 @@
+import {Key, ElementFinder} from 'protractor';
+
+import {openUrl, expectFocused, sendKey, getCaretPosition} from '../../tools.po';
+
+import {Page} from './timepicker-navigation.po';
+
+describe('Timepicker', () => {
+  let page: Page;
+
+  beforeAll(() => page = new Page());
+  beforeEach(async() => await openUrl('timepicker/navigation'));
+
+  function focusInputBefore() { return page.getInputBefore().click(); }
+
+  function goNext() { return sendKey(Key.TAB); }
```

Comment:
I would argue that these keyboard helpers make it MORE difficult to read the test.

I would prefer to read the test:

```
focus input before
press TAB
expect hourField to be focused
press SHIFT+TAB
...
```

as opposed to using `goNext()` which I have no idea what it does.
What's the point of hiding of what you're actually doing and wrapping one function into another here?

P.S. talking about only `sendKey(...)` here